test/system/business_profile_test.rb

```diff
@@ -0,0 +1,14 @@
+
+require "application_system_test_case"
+
+class BusinessProfileTest < ApplicationSystemTestCase
```

Comment:
@joemasilotti When I run this system test for some reason it's running on port 3000 so it's connecting to my dev server instead of the test server.

Any thoughts on why that might be and how to fix it?

Reply:
This is how I'm running the test

`rails test test/system/business_profile_test.rb HEADFUL=1`

Reply:
Hmm, I'm not sure. They are working fine for me.

But honestly, I don't think we need a system test for this. There's no custom JavaScript on this page so it feels like overkill. Adding to the existing integration test should be good enough.

Reply:
> Adding to the existing integration test should be good enough

Which test are you referring to?

Reply:
`test/integration/businesses_test.rb`

Reply:
I have updated the existing integration tests and removed my system test.

https://github.com/joemasilotti/railsdevs.com/pull/397/commits/69fa0632cc9c58f7c7f3fb36015e53ed7c11590b